### ISSUE TITLE
Directory listing broken

### ISSUE THREAD

**Issue by kiwi2301** [Arlen22/TiddlyServer#45]
Hi, thanks for the good work. I've upgraded to 2.0.14 today, and I might have broken something... I cannot be sure if this happened in 2.0.14 or 2.0.13, because I've set my bookmark to direct to my wiki previously. Did a clean install today and found out. The directory listing looks like this now:

![Directory Listing 1](https://i.imgur.com/uTO1lKK.png)

![Directory Listing 2](https://i.imgur.com/Nb4IijF.png)

I can no longer upload files or create subdirectories and data folders. It also seems that the table styling in directory.css doesn't work as well, but the browser loaded the stylesheet fine.

I'm running TiddlyServer on Debian 9, Nodejs 10.4.1 behind Nginx. Uploading a data folder with rsync and pointing to it in settings.json works fine, just that the listing is broken. I know it worked when I first installed it, so that'll be 2.0.12. I really can't remember if it worked when I upgraded to 2.0.13, sorry.

I'm not proficient in Linux yet, would really appreciate some help. Thank you in advance!

**Comment by Arlen22** [comment]
Yes, I modified the styles because I found the table to be too cumbersome on mobile. The new look felt cleaner, although I still need to make sure all the icons are set correctly. 

Please refer to the settings docs for the upload and create directory permissions at https://github.com/Arlen22/TiddlyServer/blob/master/SETTINGS.md#allownetwork-and-allowlocalhost

The release that lists this change is https://github.com/Arlen22/TiddlyServer/releases/tag/2.0.13. Unfortunately it doesn't give much detail but the settings were added to allow people in different network environments to set whether network users were allowed to do more "power user" type stuff.

For a complete example of all the possible keys in `settings.json`, see the Typescript interface definition at https://github.com/Arlen22/TiddlyServer/blob/master/SETTINGS.md#settingsjson.
